front/app/components/FormBuilder/edit/index.tsx

```diff
@@ -109,14 +120,28 @@ export const FormEdit = ({
     setError,
     handleSubmit,
     control,
-    formState: { isSubmitting, errors },
+    formState: { errors },
+    reset,
   } = methods;
 
   const { append, move, replace } = useFieldArray({
     name: 'customFields',
     control,
   });
 
+  // This tracks form update. We isolate it to avoid setting data on other changes
```

Comment:
I'm sure there won't be any issues, but just checking that the ideation input form has been tested too